rest_framework/serializers.py

```diff
@@ -1081,7 +1081,10 @@ def save_object(self, obj, **kwargs):
                     self.save_object(related)
                 else:
                     # Reverse FK or reverse one-one
-                    setattr(obj, accessor_name, related)
+                    try:
+                        setattr(obj, accessor_name, related)
+                    except ValueError:
+                        getattr(obj, accessor_name).add(*related, bulk=False)
```

Comment:
Thats why I am searching for proper solution, if this is M2M relationship, use `getattr(obj, accessor_name).add(*related, bulk=False)`, if it is other relationship do `setattr(obj, accessor_name, related)`. This way we use default old behaviour and DRF is compatibile with Django 1.9.

I have no idea why they changed default behaviour to use bulk update, probably perfomance gains. 
Still, this breaks not only our codebase, but everyone else also.